.github/workflows/image.yaml

```diff
@@ -115,16 +117,13 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Build image
         env:
-          IMAGE_NAME: ghcr.io/${LOWERCASE_REPO_OWNER}/container-toolkit
-          VERSION: ${COMMIT_SHORT_SHA}
+          IMAGE_NAME: ghcr.io/nvidia/container-toolkit
+          VERSION: ${{ inputs.version }}
+          PUSH_ON_BUILD: ${{ inputs.push_on_build }}
```

Comment:
What about?
```suggestion
          PUSH_ON_BUILD: true
```

Reply:
I was going for reusability of the workflow file, but default to true is ok

Reply:
If needed, we can add `push_on_build` to the `inputs` *and* set a default value.